src/javascript/JContent/ContentTree/ContentTree.jsx

```diff
@@ -77,8 +78,15 @@ const ItemComponent = ({children, node, item, treeEntries, ...props}) => {
         }
     }, [depth]);
 
+    useEffect(() => {
+        if (ref.current) {
+            virtualizer.measureElement(ref.current);
+        }
+    }, [ref, virtualizer]);
```

Comment:
dependency here needs to be `ref.current`

```suggestion
    }, [ref.current, virtualizer]);
```

Reply:
Discussed with @AKarmanov as it was throwing lint error: `React Hook useEffect has an unnecessary dependency: 'ref.current'`.

We'll keep it back to just ref so as to trigger useEffect when ref is initialized